packages/next-server/lib/router/router.ts

```diff
@@ -56,6 +56,7 @@ export default class Router implements BaseRouter {
   clc: ComponentLoadCancel
   pageLoader: any
   _bps: BeforePopStateCallback | undefined
+  _prevUrl?: string
```

Comment:
This should be marked private right?
```suggestion
  private _prevUrl?: string
```

Reply:
This is removed now, reverted to working fix